<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>787</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1393</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="frontPage_0">
     <widget class="QLabel" name="mainTitle">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>691</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Wholesale Club&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>130</y>
        <width>421</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="salesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Sales Reports</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inventoryButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Inventory Management</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="memberButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Member Management</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="membershipButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Membership Management</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="purchasesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Purchases Management</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="salesReports_1">
     <widget class="QPushButton" name="returnButton_sales">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="salesReports_label">
      <property name="geometry">
       <rect>
        <x>404</x>
        <y>40</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Sales Reports&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="salesReportsDisplay">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>121</y>
        <width>1041</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>710</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="dayLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Day:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="daySales_select">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>620</y>
        <width>441</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="dailySalesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Daily Sales</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yearlySalesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Yearly Sales</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rebatesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Rebates</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="inventory_3">
     <widget class="QPushButton" name="returnButton_inventory">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="inventoryDisplay">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>1081</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="inventoryLabel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>521</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Inventory Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="allItemsSold_button">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>670</y>
        <width>441</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Display All Items Information</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>670</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="itemSearch_label">
           <property name="text">
            <string>Item Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="itemSearch_box"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="soloItemSold_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Display Item Information</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="members_2">
     <widget class="QPushButton" name="returnButton_members">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="memberDisplay">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>100</y>
        <width>521</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="memberTitle">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Member Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addMember_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>260</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Member</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>350</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="delete_nameRadio">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="delete_IDRadio">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deleteMember_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="deleteMember_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>390</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete Member</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>180</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="addExp_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Membership Ends:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="addExp_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>140</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="addID_label">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="addID_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>100</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="addName_label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addName_box"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>610</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="convertToBasic_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Convert to Basic List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="convertToPreferred_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Convert to Preferred List</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="allMembers_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>480</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Members</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QRadioButton" name="basicRadio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Basic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="preferredRadio">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Preferred</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="membership_4">
     <widget class="QPushButton" name="returnButton_membership">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="membershipTitle">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Membership Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="membershipDues_display">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>431</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="monthExpirings_display">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>100</y>
        <width>401</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="membershipDues_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>670</y>
        <width>431</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Membership Dues</string>
      </property>
     </widget>
     <widget class="QPushButton" name="expirings_button">
      <property name="geometry">
       <rect>
        <x>689</x>
        <y>710</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Expirings</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>694</x>
        <y>663</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="monthLabel">
         <property name="text">
          <string>Month:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="monthBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>550</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="basicOnly_membership">
         <property name="text">
          <string>Basic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="prefOnly_membership">
         <property name="text">
          <string>Preferred</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="allMem_membership">
         <property name="text">
          <string>All Members</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="purchases_5">
     <widget class="QPushButton" name="returnButton_purchases">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>710</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="purchasesTitle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Purchases Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="purchasesDisplay">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>921</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="allMemberPurchase_button">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>655</y>
        <width>451</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>All Member Purchases</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>653</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="memberPurchase_box"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="namePurchase_option">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="idPurchase_option">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="memberPurchase_button">
         <property name="text">
          <string>Total Individual Purchases</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
